### ISSUE TITLE
Bulk Slack DMs

### ISSUE THREAD

**Issue by mutualmick** [polymorphic-labs/bulk-slack-dms-extension#1]
I loaded the extension, secured a license, launched Bulk Slack DMs and created a csv file to upload. I tied to upload but it locks at that point. Am I missing something?

**Comment by noitcudni** [comment]
What do you mean that it locks at that point?  As in it's frozen?

**Comment by mutualmick** [comment]
Sorry, yes, the upload does not occur and I am not allowed to send a message.
Mick
    On Wednesday, March 13, 2024 at 01:10:28 PM CDT, Lih ***@***.***> wrote:  
 
 


What do you mean that it locks at that point? As in it's frozen?

**Comment by noitcudni** [comment]
I see, so the file dialog doesn't open at all?
Which operating system and chrome version are you using? I'll try to reproduce it

**Comment by mutualmick** [comment]
Thanks. Chrome is up to dateVersion 122.0.6261.129 (Official Build) (64-bit)Windows 10 Home v22H2, OS build 19045.4170 with Windows Feature Experience Pack 1000.19054.1000.0
Mick Occhiuto 

    On Wednesday, March 13, 2024 at 01:55:49 PM CDT, Lih ***@***.***> wrote:  
 
 


I see, so the file dialog doesn't open at all?
Which operating system and chrome version are you using? I'll try to reproduce it

**Comment by noitcudni** [comment]
Would you do me a favor? Can you right click anywhere on the page and select inspect to bring up the developer console.
Under the elements tab, can you search for `memberFileInput`? See screenshot
<img width="952" alt="image" src="https://github.com/polymorphic-labs/bulk-slack-dms-extension/assets/78951/76eb6759-e92e-4808-908a-9650d782d7a2">
Ctrl - F to find.

**Comment by noitcudni** [comment]
Also, one other thing you can try.
Click on the console tab in your developer console.
Type the following code snippet
```$x("//input[@id='memberFileInput']")[0].click()```

See if the dialog pops up.

**Comment by mutualmick** [comment]
Two hits on ...


memberFileInput



Mick OcchiutoVice President - MembershipBellevue Bicycle Club
***@***.*** 

    On Wednesday, March 13, 2024 at 09:05:09 PM CDT, Lih ***@***.***> wrote:  
 
 


Would you do me a favor? Can you right click anywhere on the page and select inspect to bring up the developer console.
Under the elements tab, can you search for memberFileInput? See screenshot
image.png (view on web)

**Comment by mutualmick** [comment]
It does bring up a browse of my computer. I selected the csv file but the same thing happens. The csv file doesn't load. Could it have to do with the csv file format?

***@***.***='memberFileInput']")[0].click()



Mick Occhiuto
    On Thursday, March 14, 2024 at 12:48:40 AM CDT, Lih ***@***.***> wrote:  
 
 


Also, one other thing you can try.
Click on the console tab in your developer console.
Type the following code snippet
***@***.***='memberFileInput']")[0].click()

See if the dialog pops up.

**Comment by noitcudni** [comment]
Can you email me your csv file? lihster.developer@gmail.com
I'm not suggesting that as a solution. I'm just trying to narrow down the problem space.

**Comment by noitcudni** [comment]
Also, can you right click anywhere inside the extension and select inspect to bring up your developer console.

Click on the console tab. Reload the extend and take a screenshot. It should look like this.
<img width="943" alt="image" src="https://github.com/polymorphic-labs/bulk-slack-dms-extension/assets/78951/02087066-3816-4a6a-99ed-72f9ada2c506">

Then, inside the same developer console, click on the network tab and select js and take another screenshot and send that to me. It should look like this.

<img width="954" alt="image" src="https://github.com/polymorphic-labs/bulk-slack-dms-extension/assets/78951/f3088e84-9122-4396-b10d-491deef6f472">

**Comment by mutualmick** [comment]
Mick 

    On Thursday, March 14, 2024 at 10:32:54 PM CDT, Lih ***@***.***> wrote:  
 
 


Can you email me your csv file? ***@***.***
I'm not suggesting that as a solution. I'm just trying to narrow down the problem space.

**Comment by mutualmick** [comment]
Mick 

    On Thursday, March 14, 2024 at 10:38:35 PM CDT, Lih ***@***.***> wrote:  
 
 


Also, can you right click anywhere inside the extension and select inspect to bring up your developer console.

Click on the console tab. Reload the extend and take a screenshot. It should look like this.
image.png (view on web)

Then, inside the same developer console, click on the network tab and select js and take another screenshot and send that to me. It should look like this.
image.png (view on web)

**Comment by mutualmick** [comment]
This issue was never resolved. Your support team just went silent while we were working on it.
Mick OcchiutoVice President - MembershipBellevue Bicycle Club
***@***.*** 

    On Tuesday, May 7, 2024 at 08:48:14 PM CDT, Lih ***@***.***> wrote:   

 


Closed #1 as completed.

**Comment by noitcudni** [comment]
I can keep it open. Please answer my last message.

**Comment by mutualmick** [comment]
This is the last message I recall receiving and my response sent to you March 15.
Mick OcchiutoVice President - MembershipBellevue Bicycle Club
***@***.*** 

   ----- Forwarded Message ----- From: Mick Occhiuto ***@***.***>To: polymorphic-labs/bulk-slack-dms-extension ***@***.***>Sent: Friday, March 15, 2024 at 07:31:09 AM CDTSubject: Re: [polymorphic-labs/bulk-slack-dms-extension] Bulk Slack DMs (Issue #1)  
 



Mick 

    On Thursday, March 14, 2024 at 10:38:35 PM CDT, Lih ***@***.***> wrote:  
 
 


Also, can you right click anywhere inside the extension and select inspect to bring up your developer console.

Click on the console tab. Reload the extend and take a screenshot. It should look like this.
image.png (view on web)

Then, inside the same developer console, click on the network tab and select js and take another screenshot and send that to me. It should look like this.
image.png (view on web)

**Comment by noitcudni** [comment]
Ah, I see. you have to go into github directly and upload the images. I think you tried to paste the images from your email. Github striped them and showed only "image.png (view on web)".

**Comment by noitcudni** [comment]
Also, can you send me your csv file?

**Comment by mutualmick** [comment]
I sent that on March 14th. Seems like you would have said you just got an empty email. Here it is again. Please confirm you received
[slackcsv.csv](https://github.com/polymorphic-labs/bulk-slack-dms-extension/files/15251389/slackcsv.csv)
 it.

**Comment by mutualmick** [comment]
Here is the graphics from the Mar 15 response. Let me know you received.
![Screenshot 2024-03-15 072733](https://github.com/polymorphic-labs/bulk-slack-dms-extension/assets/35844354/83d61ce7-99b4-4c96-8d72-785bef780257)
![Screenshot 2024-03-15 073011](https://github.com/polymorphic-labs/bulk-slack-dms-extension/assets/35844354/a782d970-54b4-4f38-833c-a8b5f044c357)

**Comment by noitcudni** [comment]
Yeah, I think you tried attaching the file from your email. Github stripped it.

Instead of having first and last name, replace them with users email address.

**Comment by mutualmick** [comment]
Did you get them? I went directly to Github and uploaded them there.
Mick OcchiutoVice President - MembershipBellevue Bicycle Club
***@***.*** 

    On Wednesday, May 8, 2024 at 10:54:26 AM CDT, Lih ***@***.***> wrote:   

 


Yeah, I think you tried attaching the file from your email. Github stripped it.

Instead of having first and last name, replace them with users email address.

**Comment by noitcudni** [comment]
Yes, I can see them, but did you see my last message?
```
Instead of having first and last name, replace them with users email address.
```
^^  Try that

**Comment by mutualmick** [comment]
Oh! I thought you were still trying to explain to me how to send a message to you. You mean in the CSV file just use one column with email address.  I'll try that and let you know. That wasn't clear in any written explanation.
Mick OcchiutoVice President - MembershipBellevue Bicycle Club
***@***.*** 

    On Wednesday, May 8, 2024 at 12:51:27 PM CDT, Lih ***@***.***> wrote:   

 


Yes, I can see them, but did you see my last message?
Instead of having first and last name, replace them with users email address.

^^ Try that

**Comment by noitcudni** [comment]
Yeah, I'll work on adding better instructions tonight.

**Comment by mutualmick** [comment]
That worked great! Thank you. Definitely a key instruction for the average Joe that needs to be included. Also, although I "kind of" know what you mean here:
One of these arguments (attachments/blocks/text) is required to describe the content of the message. If attachments or blocks are included, text will be used as fallback text for notifications only.If you'd like to format the dm content as attachments or blocks, please verify the json payload in Slack's JSON Builder.

I think I would have that written in average personeese too.  Something like:
At least one of these fields (attachments/blocks/text) is required. If you would like to format the direct message content as attachments or blocks, please verify the JSON in Slack's JSON Builder. The text field will be the default if attachments or blocks are included for notifications only.
I may not fully understand that instruction but I think that's what you're trying to say.

Mick OcchiutoVice President - MembershipBellevue Bicycle Club
***@***.*** 

    On Wednesday, May 8, 2024 at 01:16:33 PM CDT, Lih ***@***.***> wrote:   

 


Yeah, I'll work on adding better instructions tonight.

**Comment by noitcudni** [comment]
Thanks for the feedback. I'll work on making the instructions more clear. 
If you don't mind, please leave a positive review here: https://chromewebstore.google.com/detail/bulk-slack-dms/lmmmbcdpeakolpppakanidkbfaflkcie. It will help alot.

**Comment by noitcudni** [comment]
Thank you for your kind review.
